server/views/components/feedback-widget/template.njk

```diff
@@ -30,25 +55,42 @@
       class="govuk-hub-comment-form govuk-u-hidden"
     >
       <div class="govuk-!-margin-bottom-2">
-        <p id="more-detail" class="govuk-hint">
-          <a href="/help" class="govuk-link govuk-link--no-visited-state">Click here if you need help and are struggling with mental health.</a><br>You will not get the help you need if you leave a message in the box below.
-        </p>
-        <p class="govuk-body">We get a lot of feedback but try to read as much as we can. Feedback is treated anonymously, unless it breaks prison rules or raises safety concerns.</p>
-        <textarea data-feedback-comment class="govuk-textarea govuk-!-margin-bottom-0 govuk-!-padding-2" id="more-detail" name="value" rows="4" placeholder="I think the Hub should:" maxlength="240" required></textarea>
-      </div>
-      <div class="govuk-hub-comment-form__footer">
-        <div class="govuk-hub-comment-counter">
-          <span data-feedback-comment-counter >240</span> characters left</div>
-        <button type="submit" class="govuk-button">Send</button>
+        {% set feedbackLikeTexts = [
+          "This content made a positive difference to me",
+          "I’d like more of this kind of content"
+        ] %}
+        {{ feedbackOptions(feedbackLikeTexts, 'feedbackOption-LIKE') }}
+        {% set feedbackDislikeTexts = [
+          "Content is missing",
+          "Broken link on this page",
+          "Audio or video isn’t playing",
+          "Images aren’t working",
+          "There is wrong information on this page"
```

Comment:
shouldnt 'wrong' be 'incorrect' ?

Reply:
Agreed. I'll raise with the content team 👍